MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationCameraController.java

```diff
@@ -355,6 +356,17 @@ private void notifyCameraTrackingChangeListener(boolean wasTracking) {
     }
   }
 
+  private MapboxMap.OnCameraMoveListener onCameraMoveListener = new MapboxMap.OnCameraMoveListener() {
+
+    @Override
+    public void onCameraMove() {
+      if (isLocationTracking()) {
+        adjustFocalPoint = true;
```

Comment:
Could the logic that resets the focal point be moved to the camera move listener entirely? This would allow us to clean up the class and not maintain the `adjustFocalPoint` flag and the `lastLocation` object.